data-sets-tests/src/test/java/org/zowe/unix/files/tests/UnixFilesDeleteFileTest.java

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
For consistency with other integration tests and for readability when reviewing test failures down the line rename file to UnixFilesDeleteFileIntegrationTest.java

Reply:
modified added "integration" to name